### ISSUE TITLE
Error: Call to undefined function dpm()

### ISSUE THREAD

**Issue by olafgrabienski** [Graham-72/beautytipsX#2]
When I try to save the configuration on the page `admin/config/content/beautytips`, I see a nearly blank page telling me "The website encountered an unexpected error". In the DBlog there is a corresponding message:

> Error: Call to undefined function dpm() in beautytips_admin_submit() (line 187 of /(...)/modules/beautytips-1.x-2.x/beautytips.admin.inc).

Maybe related: "dpm" was removed in commit https://github.com/Graham-72/beautytips/commit/d7f86f5a9b438ff00136e49eaaab70a1f8623c92.

**Comment by Graham-72** [comment]
Thanks for reporting this. There was another dpm and so I have made another commit to remove it.

**Comment by olafgrabienski** [comment]
That did the trick, perfect! Now I've been able to check the first `beautytips js` setting and to save the configuration page. A popup using a respective element works.

(Btw, first I got the same error as mentioned in the issue description, then I realized that the other 'dpm' wasn't removed, so I removed it manually in the respective file. I guess the download link to https://github.com/Graham-72/beautytips/archive/1.x-2.x.zip must have been cached.)
